README.md

```diff
@@ -31,26 +31,30 @@ Starting with a bare machine ..
 ### Agent deployment keys ###
 
 Hosts are configured so that login as root is by means of a deploy key.
-The canonical source for the deploy key is the DevOps secret store.
 You must have the deploy key available to your local SSH agent before you can
 log into the box or run the playbook. To add the deploy key to your local SSH
 agent:
 
 ```bash
-$ cd ~/repos/devops/docs/secrets  # location of secrets store clone
+$ cd secrets
 $ alias vault="ansible-vault --vault-password-file=$PWD/open-vault"
```

Comment:
Could we just have ``alias vault="ansible-vault --vault-password-file=$PWD/secrets/open-vault"`` to match the command below and update the examples to use, e.g. "secrets/lecture-capture-agent-deploy-main"?